roles/sap_swpm/README.md

```diff
@@ -63,58 +63,66 @@ This role has been tested and working for the following SAP products
+    - Wrong parameters will result in failure
 
 Create an input file which contains all relevant installation information.
-Sample input files are stored in the [inputs](/playbooks/vars) folder of this Ansible collection. Use the samples as guide for your desired installation
-
-### Default Parameters
+Sample input files are stored in the [inputs](/playbooks/vars) folder of this Ansible collection. Use the samples as guide for your desired installation.
 
 ### Default Parameters
 
 Please check the default parameters file for more information on other parameters that can be used as an input:
 - [**sap_swpm** default parameters](defaults/main.yml)
 
-Sample Playbooks and sample parameters are shown in the Ansible Collection `/playbooks` directory.
-
-The Ansible Collection `/playbooks` directory includes sample playbooks for using the `sap_swpm` Ansible Role in the following 'modes':
-- Default
-- Default Templates
-- Advanced
-- Advanced Templates
-- Inifile Reuse
-
-The Ansible Collection `/playbooks/vars` directory includes sample variable files for:
-- `sap_swpm` 'Default mode' to generate inifile.params of...
-  - SAP BW/4HANA OneHost
-  - SAP S/4HANA Distributed - ASCS, DBCI, ERS, PAS
-  - SAP S/4HANA OneHost
-  - SAP S/4HANA System Copy OneHost
-  - SAP Solution Manager (ABAP)
-  - SAP Solution Manager (JAVA)
-  - SAP Web Dispatcher
-  - SAP System Rename
-- `sap_swpm` 'Default Templates mode' to generate inifile.params of...
-  - SAP S/4HANA OneHost
-  - SAP S/4HANA System Copy OneHost
-  - SAP System Rename
-- `sap_swpm` 'Advanced mode' to generate inifile.params of...
-  - SAP S/4HANA OneHost
-- `sap_swpm` 'Advanced Templates mode' to generate inifile.params of...
-  - SAP BW/4HANA OneHost
-  - SAP S/4HANA Distributed - ASCS, DBCI, ERS, PAS
-  - SAP S/4HANA OneHost
-  - SAP S/4HANA System Copy OneHost
-  - SAP Solution Manager (ABAP)
-  - SAP Solution Manager (JAVA)
-  - SAP Web Dispatcher
-  - SAP System Rename
-- `sap_swpm` 'Inifile Reuse mode' inifile.params file for...
-  - SAP S/4HANA OneHost
-
-NOTE: these are only sample files, they are meant to be edited by the user before execution and do not cover all scenarios possible (the Ansible Role can execute ant SAP SWPM installation)
+### Migrating playbooks from previous versions of sap_swpm
+The following role parameter is no longer used because there are no role `modes` any more:
+#### sap_swpm_ansible_role_mode
+
+The following two role parameters have been renamed, without automatic conversion between old and new name:
+
+#### sap_swpm_inifile_list -> sap_swpm_inifile_sections_list
+Previous name: sap_swpm_inifile_list
+New name: sap_swpm_inifile_sections_list
+Reason: This variable contains sections of the sapinst input file, `inifile.params`.
+The new variable name is reflecting this purpose.
+
+#### sap_swpm_inifile_custom_values_dictionary -> sap_swpm_inifile_parameters_dict
+Previous name: sap_swpm_inifile_custom_values_dictionary
```

Comment:
</br> is missing in these lines resulting in confusing one line.

Reply:
What do you think is missing, and where?

Reply:
updated my comment as it was considered part of markdown

Reply:
So you suggest adding an empty line after all `###` and `####` sections which do not have one?

Reply:
Yes, because it seems that intent was to have clean lines based on readme file.

```markdown
#### sap_swpm_inifile_list -> sap_swpm_inifile_sections_list
Previous name: sap_swpm_inifile_list
New name: sap_swpm_inifile_sections_list
Reason: This variable contains sections of the sapinst input file, `inifile.params`.
The new variable name is reflecting this purpose.
```

But actual Readme preview shows correctly that markdown does not do newline automatically
```markdown
sap_swpm_inifile_list -> sap_swpm_inifile_sections_list
Previous name: sap_swpm_inifile_list New name: sap_swpm_inifile_sections_list Reason: This variable contains sections of the sapinst input file, inifile.params. The new variable name is reflecting this purpose.
```

Reply:
@marcelmamula I now see what you mean after previewing the README.md file. The lines below he caption are concatenated but they should remain separate.